controller/sync_strats/rtx.go

```diff
@@ -73,15 +73,15 @@ func (rtx *RouterSender) Start() {
 func (rtx *RouterSender) Stop() {
 	if rtx.stopping.CompareAndSwap(false, true) {
 		go func() {
```

Comment:
Usual pattern is to just close the `closeNotify` channel. Also means you shouldn't have to do it in a separate goroutine.

Reply:
Changed